### ISSUE TITLE
It's a bug?

### ISSUE THREAD

**Issue by zhangjf108** [Gottox/socket.io-java-client#25]
My server side is both hold socket.io and http, when i run this, it just visit the http server?

**Comment by Gottox** [comment]
Hi!

Can you rephrase your question? I don't understand your question/bugreport.

kind regards

Enno

**Comment by zhangjf108** [comment]
Both http and socket(socket.io) service run on my server side,and they listen the same port. When I run the socket.io java client, it just vist access the http server~thx~

**Comment by Gottox** [comment]
Are you sure it is a client issue? Does it work with a javascript-client?
